server-phoenix/lib/helios_web/schema/types/weather.ex

```diff
@@ -48,14 +50,16 @@ defmodule HeliosWeb.Schema.Types.Weather do
     field(:wind_speed, non_null(:float))
     field(:wind_deg, non_null(:integer))
     field(:clouds, non_null(:integer))
-    field(:uvi, non_null(:integer))
+    field(:uvi, non_null(:float))
```

Comment:
Why did this change in ` feat: add resolver for weather_info_data `? Does it need to be in the next commit?

Reply:
Yes! I must've missed that when splitting up the commit.